### PR TITLE
updated email field to correct label

### PR DESCRIPTION
**Pull request by joeyrivera**

### CODE REVIEW

File: docs/kb/api-version-3/emails.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ For a complete list of fields involved in user operations, see the [Email](../ob
 
 | **Operation** | **URL Format**                             | **Required Parameters** | **Description**  |
 | ------------- | ------------------------------------------ | ----------------------- | -----------------|
-| `read` | `/api/email/version/3/do/read/id/<email id>?...` | `user_key, api_key, email` | Returns the data for the email specified by `<id>`. `<id>` is the Pardot ID of the target email. |
+| `read` | `/api/email/version/3/do/read/id/<email id>?...` | `user_key, api_key, email_id` | Returns the data for the email specified by `<email_id>`. `<email_id>` is the Pardot ID of the target email. |
```
Comment (juan-nunez) [inline docs/kb/api-version-3/emails.md]: This is picky but for the sake of uniformity, can you add an underscore to the email_id in /api/email/version/3/do/read/id/<email id> <--- is email id right now. Same for the other files.

**Comment by joeyrivera** [comment]
@juan-nunez updated.

**Comment by juan-nunez** [comment]
👍

File: docs/kb/release-notes.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 This page contains the release notes for the Pardot API and related documentation.
 
+## July 2017
+
+### Email
+* updated "Reading Emails" section to correctly label the required param `email_id` instead of `email`.
```
Comment (easher) [inline docs/kb/release-notes.md]: @joeyrivera Did you mean leave out changes for list_email_id here?

Comment (joeyrivera) [inline docs/kb/release-notes.md]: yeah, didn't see much more value adding that there also but can

Comment (joeyrivera) [inline docs/kb/release-notes.md]: added.

**Pull request merged**